src/cpp/fastdds/publisher/DataWriter.cpp

```diff
@@ -156,6 +156,12 @@ ReturnCode_t DataWriter::assert_liveliness()
     return impl_->assert_liveliness();
 }
 
+bool DataWriter::remove_all_changes(
+        size_t* removed)
+{
+    return impl_->remove_all_changes(removed);
+}
+
```

Comment:
Change if the method's name finally changes to `clear_history` as proposed elsewhere